taca/analysis/analysis.py

```diff
@@ -1,19 +1,20 @@
 """ Analysis methods for TACA """
 import csv
+from datetime import datetime
```

Comment:
that's a bit of a strange import organization (not saying that it wasn't before tohugh). I usually group total standard imports (`import whatever_standard_package`) at the beginning in alphabetical order, then total non-standard packages in alphabetical order as well, then relative standard imports and finally relative non-standard imports. 

Do you understand me? I am re-reading what I wrote and I'm not sure I would, haha